src/py/flwr/cli/ls.py

```diff
@@ -104,11 +119,13 @@ def ls(
             # Display information about a specific run ID
             if run_id is not None:
                 typer.echo(f"🔍 Displaying information for run ID {run_id}...")
-                _display_one_run(stub, run_id)
+                restore_output()
```

Comment:
Do we need `restore_output()` here, since we have it in `finally` block?

Reply:
Yes. This is because `Console().print_json()` will also be redirected and won't be shown.